templates/windows2k12.tpl.yaml

```diff
@@ -2,11 +2,11 @@
 apiVersion: template.openshift.io/v1
```

Comment:
All new templates you created should have `{% set version = "0.12.3" %}` as this is going to be the first version they were introduced into common-templates

Reply:
Done